spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-function/src/main/java/org/springframework/cloud/dataflow/tasklauncher/TaskLauncherFunction.java

```diff
@@ -43,34 +43,30 @@
  *
  * @author David Turanski
  **/
-public class TaskLauncherFunction implements Function<LaunchRequest, Optional<LaunchResponse>>, InitializingBean {
-	private static final Log log = LogFactory.getLog(TaskLauncherFunction.class);
+public class TaskLauncherFunction implements Consumer<LaunchRequest>, InitializingBean {
 
-	static final String TASK_PLATFORM_NAME = "spring.cloud.dataflow.task.platformName";
+	private static final Logger log = LoggerFactory.getLogger(TaskLauncherFunction.class);
 
-	private final TaskOperations taskOperations;
+	public static final String TASK_PLATFORM_NAME = "spring.cloud.dataflow.task.platformName";
```

Comment:
Why public?

Reply:
It was used in Test code for the same property.

Reply:
Ahh ok. If its in the same package then a good approach is to make it package visibility and comment it w/ 
```java
// VisibleForTesting
```

Reply:
Please update w/ the above.

Reply:
done